web/src/components/storage/ProposalSettingsSection.jsx

```diff
@@ -312,12 +318,29 @@ export default function ProposalSettingsSection({
 
   const encryption = settings.encryptionPassword !== undefined && settings.encryptionPassword.length > 0;
 
+  const transactional = isTransactionalSystem(settings?.volumes || []);
+
   return (
     <>
       <Section title={_("Settings")}>
-        <SnapshotsField
-          settings={settings}
-          onChange={changeBtrfsSnapshots}
+        <If
+          condition={transactional}
+          then={
+            <div>
+              <label>{_("Transactional system")}</label>
+              <div>
+                {/* TRANSLATORS: %s is replaced by a product name (e.g., openSUSE Tumbleweed) */}
+                {sprintf(_("%s is an immutable system with atomic updates using a read-only Btrfs \
+root file system."), selectedProduct.name)}
+              </div>
+            </div>
```

Comment:
Note for a follow-up, create a component for this kind of structure (two rows, one for explanation). This can be done when swapping the position of the switchers to have them aligned.